javascript/ql/src/semmle/javascript/PackageExports.qll

```diff
@@ -78,5 +78,12 @@ private DataFlow::Node getAValueExportedByPackage() {
 private DataFlow::Node getAnExportFromModule(Module mod) {
   result.analyze().getAValue() = mod.(NodeModule).getAModuleExportsValue()
   or
+  exists(Variable var | var = mod.(Closure::ClosureModule).getExportsVariable() |
```

Comment:
Do we have a test for this Closure case?

Reply:
Nope. I'll get on it.